renderer/containers/UI/FormattedDateTime.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { FormattedDate, FormattedTime } from 'react-intl'
+import { settingsSelectors } from 'reducers/settings'
+
+const mapStateToProps = state => ({
+  timeDisplayMode: settingsSelectors.currentConfig(state).timeDisplayMode,
+})
+
+const FormattedDateTime = ({ timeDisplayMode, format, month = 'long', value }) => {
```

Comment:
Probably you should set default here: `timeDisplayMode = '12hour'`

`value` is required so maybe bring it to the start of the prop list.

Reply:
Have set default timeDisplayMode to '12hour'
linter looks for alphabetical order of prop list, so have sorted accordingly.